integration-framework/src/main/java/org/apache/cassandra/sidecar/testing/SharedClusterIntegrationTestBase.java

```diff
@@ -350,6 +351,9 @@ protected Server startSidecarWithInstances(Iterable<? extends IInstance> instanc
         AbstractModule testModule = new IntegrationTestModule(instances, classLoaderWrapper, mtlsTestHelper,
                                                               dnsResolver, configurationOverrides());
         sidecarServerInjector = Guice.createInjector(Modules.override(new MainModule()).with(testModule));
+        Vertx vertx = sidecarServerInjector.getInstance(Vertx.class);
+        vertx.eventBus()
+             .localConsumer(SidecarServerEvents.ON_SIDECAR_SCHEMA_INITIALIZED.address(), msg -> sidecarSchemaReadyLatch.countDown());
```

Comment:
register the event consumer at the earliest. The original setup misses the event occasionally.